lib/new_relic/util.ex

```diff
@@ -20,6 +20,10 @@ defmodule NewRelic.Util do
     end
   end
 
+  def get_req_header(%{req_headers: headers}, key) when is_binary(key) do
+    for {^key, value} <- headers, do: value
```

Comment:
I've not seen this form of key access before. Is it so it returns a list?

Reply:
Is this the same as doing: `headers[key] |> List.wrap()`?

Reply:
I just see a lot of `Util.get_req_header(conn, "referer") |> List.first()`, usage.

Reply:
Or maybe there can be N values with the same key …?